docs/install/install-jetson.md

```diff
@@ -128,12 +130,13 @@ To use this with the MXNet Python binding, you must match the source directory's
 ```bash
 cd $MXNET_HOME
 git checkout v1.4.x
-git submodule update --init
+git submodule update --init --recursive
 cd python
 sudo pip install -e .
 ```
```

Comment:
I think we considered adding a note here at the end of Option 1 that notifies the user that at this point, they are done and do not need to continue. I think that still sounds helpful, but that may already be clear with "Option" now in the title.